conf/tachyon-env.sh.template

```diff
@@ -83,8 +107,17 @@ export TACHYON_JAVA_OPTS+="
   -Dtachyon.worker.data.folder=/tachyonworker/
   -Dtachyon.master.max.worker.threads=$TACHYON_MASTER_MAX_WORKER_THREADS
   -Dtachyon.master.worker.timeout.ms=60000
-  -Dtachyon.master.hostname=$TACHYON_MASTER_ADDRESS
   -Dtachyon.master.journal.folder=$TACHYON_HOME/journal/
+  -Dtachyon.master.bind.host=$TACHYON_MASTER_BIND_HOST
+  -Dtachyon.master.web.bind.host=$TACHYON_MASTER_WEB_BIND_HOST
+  -Dtachyon.worker.bind.host=$TACHYON_WORKER_BIND_HOST
+  -Dtachyon.worker.data.bind.host=$TACHYON_WORKER_DATA_BIND_HOST
+  -Dtachyon.worker.web.bind.host=$TACHYON_WORKER_WEB_BIND_HOST
+  -Dtachyon.master.hostname=$TACHYON_MASTER_HOSTNAME
+  -Dtachyon.master.web.hostname=$TACHYON_MASTER_WEB_HOSTNAME
+  -Dtachyon.worker.hostname=$TACHYON_WORKER_HOSTNAME
+  -Dtachyon.worker.data.hostname=$TACHYON_WORKER_DATA_HOSTNAME
+  -Dtachyon.worker.web.hostname=$TACHYON_WORKER_WEB_HOSTNAME
```

Comment:
By default there are no changes here, and if the user wants to set them, they will add the necessary lines here or in the site properties file which will then override the defaults.